e2e_tests/tests/per_provider/normal_tests/export_public_key.rs

```diff
@@ -1,14 +1,13 @@
 // Copyright 2019 Contributors to the Parsec project.
 // SPDX-License-Identifier: Apache-2.0
-#[cfg(any(feature = "mbed-crypto-provider", feature = "cryptoauthlib-provider"))]
+#![allow(unused_imports, unused)]
```

Comment:
I've swapped the explicit `cfg` for features tagged on inports and constants with this because it was starting to be a mess of figuring out what's used where, and trying to align across tens of tests that are supported by different providers.

Reply:
Sure, that's totally fine for tests